tests/nnstreamer_grpc/runTest.sh

```diff
@@ -18,6 +18,13 @@ fi
 # This is compatible with SSAT (https://github.com/myungjoo/SSAT)
 testInit $1
 
+# Skip test on qemu env.
+if [[ "$SKIP_QEMU_ARM64_INCOMPATIBLE_TESTS" == "1" ]]; then
```

Comment:
How about deciding whether to perform the test in the shell?
```
arch=$(uname -m)
if [ "$arch" = "aarch64" ] ; then
...
```

Reply:
Well I think this approach is much easier to understand the trouble.
if we get similar problems in future, let's discuss about generalizing this.